File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let answer = "";
    for(let i = 0; i < s.length; i++){
        if(s[i] == ' '){
            answer += " ";
            continue;
        }
        const z = s[i] === s[i].toUpperCase() ? 'Z' : 'z';
        if(s.charCodeAt(i) + n > z.charCodeAt(0)){
            answer += String.fromCharCode(s.charCodeAt(i) - 26 + n);
        }
        else answer += String.fromCharCode(s.charCodeAt(i) + n);
    }
    return answer;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        String answer = "";
        for(int i = 0; i < s.length(); i++){
            char z = 'z';
            if(s.charAt(i) == ' '){
                answer += " ";
                continue;
            }
            z = Character.isUpperCase(s.charAt(i)) ? 'Z' : 'z';
            if(s.charAt(i) + n > z){
                answer += (char)((s.charAt(i) - 26) + n);
            }
            else answer += (char)(s.charAt(i) + n);
        }
        return answer;
    }
}